.travis.yml

```diff
@@ -17,6 +16,9 @@ before_install:
   - "export DISPLAY=:99.0"
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
+install:
```

Comment:
I think you should add `npm install -g npm` (or equivalent) to upgrade npm before installation. I've seen a lot of bugs with older npms on 4.x around dependencies.